### PR TITLE
byzanz-record: add page

### PR DESCRIPTION
**Pull request by tricantivu**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 0.3

I did not check the first item in the list above because I am not completely sure if the `byzanz-record` command will execute properly in operating systems other than Linux. I cloned the repository of `byzanz-record` (i.e.  https://gitlab.gnome.org/Archive/byzanz) and noticed some source files read X11 and/or glib headers (e.g. https://gitlab.gnome.org/Archive/byzanz/-/blob/master/src/byzanzsession.c), which might mean that those are some or all of the dependencies. 

Also, I wanted to include the following example for recording the current window:

```shell
eval $(xdotool getwindowfocus getwindowgeometry --shell) && byzanz-record --x "${X}" --y "${Y}" --width "${WIDTH}" --height "${HEIGHT}" {{path/to/file.[byzanz|flv|gif|ogg|ogv|webm]}}
```

However, I feared it might be too long and complex.

### CODE REVIEW

**Comment by gutjuri** [comment]
From the description i'd image that this program can run anywhere X.org runs, e.g. the BSDs. So I'd put this page in the common folder.

File: pages/linux/byzanz-record.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# byzanz-record
+
+> Create media files from screen recordings.
```
Comment (gutjuri) [inline pages/linux/byzanz-record.md:3]:
```suggestion
> Record the screen.
```

File: pages/linux/byzanz-record.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://linux.die.net/man/1/byzanz-record>.
+
+- Record the screen and write the recording to a file (by default, byzanz-record will only record for 10 seconds):
+
+`byzanz-record {{path/to/file.[byzanz|flv|gif|ogg|ogv|webm]}}`
+
+- Send a desktop notification after recording:
```
[the remark this answers is not in the document]
Comment (tricantivu) [reply pages/linux/byzanz-record.md:10]: I did not understand what you meant by "`echo`-command", could you please elaborate?

**Review by gutjuri — APPROVED** [review]

File: pages/linux/byzanz-record.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (tricantivu) [reply pages/linux/byzanz-record.md]: I have replaced the example for one that shows a message indicating recording has finished without other commands.

File: pages/linux/byzanz-record.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# byzanz-record
+
+> Record the screen.
+> More information: <https://linux.die.net/man/1/byzanz-record>.
```
Comment (kbdharun) [inline pages/linux/byzanz-record.md:4]:
```suggestion
> More information: <https://manned.org/byzanz-record>.
```

**Review by kbdharun — APPROVED** [review]
Welcome to tldr and thanks for your contribution. I have a minor suggestion for this page after which the PR is GTG.

For more information link, we prefer linking to the author's documentation or manned (Ref. https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#more-information-links).

**Comment by kbdharun** [comment]
**Note**: The reason why I am closing and reopening the PR is to retrigger the frozen waiting for CLA status check.

**Pull request merged**